lib/geoengineer/environment.rb

```diff
@@ -56,6 +56,7 @@ class GeoEngineer::Environment
   def initialize(name, &block)
     @name = name
     @outputs = []
+    @providers = []
```

Comment:
ids are not unique, most provider ids will be 'aws' it is annoying, provider id and alias need to be unique, unless the default provider then there is no alias.